Update mx6_cubox-i to use CONFIG_SYS_GENERIC_BOARD, as
arch/<arch>/lib/board.c is deprecated.

See doc/README.generic-board for more details.

Index: u-boot/include/configs/mx6_cubox-i.h
===================================================================
--- u-boot.orig/include/configs/mx6_cubox-i.h
+++ u-boot/include/configs/mx6_cubox-i.h
@@ -33,6 +33,8 @@
 #define CONFIG_INITRD_TAG
 #define CONFIG_REVISION_TAG
 
+#define CONFIG_SYS_GENERIC_BOARD
+
 /* Size of malloc() pool */
 #define CONFIG_SYS_MALLOC_LEN		(8 * SZ_1M)
 
